Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,40 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
```

Comment:
terminating slash `https://sites.google.com/site/gogdownloader/`

Reply:
Fixed, thanks.

However, I feel we should really introduce a `style` rule (or `audit` rule? Can’t tell apart why we have two separate style checks) to enforce that if it’s important.

Reply:
There was a proposal for one but it made me break out it hives since it seemed like too much pedantry. But I don't mind annoying you on a new formula PR about it to save @vszakats the trouble.

Reply:
Fair enough 😅